cml-tdengine/src/core/task.rs

```diff
@@ -24,6 +24,30 @@ struct TaskInfo {
     ts: DateTime<Local>,
 }
 
+fn update_train_start_time(taos: &Taos, available_status: &[&str]) {
```

Comment:
As the function is only called within the `run()` function, there is no need to abstract it out.